commit policy (#28)
RedHat-SP-Security/tests: README.md

See:
* https://github.com/psss/tmt
* https://github.com/psss/fmf

## Commit / merge policy

> Every important change to tests should undergo a review withing a pull request.

Author: ```suggestion
Every important change to tests should undergo a review within a pull request.
```

### changes without a need of review

> Small, trivial changes may be committed directly the master branch.

Reviewer: directly **to** the master branch

> Small, trivial changes may be committed directly the master branch.

Author: ```suggestion
Small, trivial changes may be committed directly to the master branch.
```

> Small, trivialy changes may be committed directly the master branch.

Author: ```suggestion
Small, trivial changes may be committed directly the master branch.
```

> Small, trivialy changes may be committed directly the master branch.

Reviewer 2: typo: trivial

> As a typical trivial change one may considere a metadata update (updated tags, adjust, ..)

Author: ```suggestion
As a typical trivial change one may consider a metadata update (updated tags, adjust, ..)
```

or typo fixes, updated to comments and formating changes.

### changes with a need of review
#### existing tests

> All changes to the test logic (changes of expected output, expected exit codes) need to be reviewd.

Author: ```suggestion
All changes to the test logic (changes of expected output, expected exit codes) need to be reviewed.
```

#### new tests
It is recommended to push new tests via PR with review as well as a check for covering all the agreed acceptance criteria is more than welcome.

#### reviewer selection

> The review can be done by any member of the team. However, it is recommended to ask the devel counterpart as they could judge the test expectations according to the actualy code change. Asking another QE person, on the other hand, will ansure to keep the test coding style as consistent as possible.

Author: ```suggestion
The review can be done by any member of the team. However, it is recommended to ask the devel counterpart as they could judge the test expectations according to the actual code change. Asking another QE person, on the other hand, will ensure to keep the test coding style as consistent as possible.
```

#### PR merge

> The marge it self should be done based on the accepted reviews. It is recommended to let the original requestor to merge the PR as they may want to do some refinements, e.g. quash some commits which were added during the review process.

Author: ```suggestion
The merge it self should be done based on the accepted reviews. It is recommended to let the original requestor to merge the PR as they may want to do some refinements, e.g. squash some commits which were added during the review process.
```

> The marge it self should be done based on the accepted reviews. It is recommended to let the original requestor to merge the PR as they may want to do some refinements, e.g. quash some commits which were added during the review process.

Reviewer 2: typos: The merge itself / e.g. squash
